contracts/network/BancorNetwork.sol

```diff
@@ -609,12 +608,9 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         if (!_networkSettings.isTokenWhitelisted(pool)) {
```

Comment:
the new whitelist var/check is completely missing

Reply:
Added